src/Forms/HTMLEditor/HTMLEditorSanitiser.php

```diff
@@ -17,8 +18,22 @@
  */
 class HTMLEditorSanitiser
 {
+    use Configurable;
     use Injectable;
 
+    /**
+     * rel attribute to add to link elements which have a target attribute (usually "_blank")
+     * this is to done to prevent reverse tabnabbing - see https://www.owasp.org/index.php/Reverse_Tabnabbing
+     * noopener includes the behaviour we want, though some browsers don't yet support it and rely
+     * upon using noreferrer instead - see https://caniuse.com/rel-noopener for current browser compatibility
+     *
+     * @var string
+     */
+    private static $link_noopener_value = 'noopener, noreferrer';
+
+    /** @var bool - allow users to opt out of this behaviour if required */
```

Comment:
Would not setting the value to `null` do this already?
I'm pretty sure that was a feature introduced with SS4 - setting config values to `null` will essentially unset them.

Reply:
might be a little confusing since I made a change so that setting _value to an empty string will remove existing noopener attributes, so to clarify...setting _value to:
a) 'noopener, noreferrer' - does what you expect
b) '' = remove any existing rel attributes
c) null disable this behaviour

I think I'd prefer to have to still have this as two configurable fields because '' and null are pretty close together since they have the same behaviour in lots of other domains e.g. truthiness

Let me know if this makes sense or if I misunderstood and you were talking about something totally different

Reply:
You've understood what I was talking about, yup.
I'm of no particularly strong opinion, to be honest. It was more of a suggestion. If @ScopeyNZ and or @robbieaverill (I see that thumbs up on the above comment) are also happy then they can merge :)

Reply:
have updated so that it follows the a) b) c) behaviour I outlined in previous comment

Reply:
Nice work, thanks :)